es6-shim.js

```diff
@@ -291,6 +291,13 @@
       return x != null && Object(x) === x;
     },
 
+    TypeIsNotObject: function (val) {
```

Comment:
Let's name this `IsPrimitive`, since that will correspond to an actual Abstract Operation